.github/workflows/rspec.yml

```
@@ -0,0 +1,53 @@
+name: "Rspec"
```

```suggestion
name: "RSpec"
```
This is the proper capitalization.